tests/test_helpers.py

```
@@ -27,87 +27,87 @@ def test_human_friendly_duration_singular_and_plural() -> None:
-    assert pyopnsense_helpers.dict_get({"a": {"b": [10, {"c": 3}]}}, "a.b") == [10, {"c": 3}]
-    assert pyopnsense_helpers.dict_get(data, "missing.path", default=5) == 5
+    assert aiopnsense_helpers.dict_get(data, "a.b.c") == 1
+    assert aiopnsense_helpers.dict_get(data, "x.1") == 1
+    assert aiopnsense_helpers.dict_get(data, "x.10", default=42) == 42
+    assert aiopnsense_helpers.dict_get({"a": {"b": [10, {"c": 3}]}}, "a.b") == [10, {"c": 3}]
+    assert aiopnsense_helpers.dict_get(data, "missing.path", default=5) == 5
 
 
 def test_timestamp_to_datetime() -> None:
     """Convert timestamp integers to datetime objects, handling None."""
     ts = int(datetime.now(tz=timezone.utc).timestamp())
```

Ruff (0.15.9)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)